src/main/java/firebase/sdk/dashboard/api/MyResource.java

```diff
@@ -0,0 +1,25 @@
+package firebase.sdk.dashboard.api;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Root resource (exposed at "myresource" path)
+ */
+@Path("myresource")
+public class MyResource {
```

Comment:
This resource is deleted in the following pr: https://github.com/googleinterns/step104-2020/pull/6